packages/sanity/src/core/releases/tool/BundlesOverview.tsx

```diff
@@ -29,7 +29,9 @@ export default function BundlesOverview() {
     () =>
       data?.reduce<{open: BundleDocument[]; archived: BundleDocument[]}>((groups, bundle) => {
         const group =
-          bundle.publishedAt && isBefore(bundle.publishedAt, new Date()) ? 'archived' : 'open'
+          bundle.publishedAt && isBefore(new Date(bundle.publishedAt), new Date())
+            ? 'archived'
+            : 'open'
```

Comment:
Just spoke to [Marius](https://sanity-io.slack.com/archives/C077WKSKVKK/p1719993779923189?thread_ts=1719991022.339779&cid=C077WKSKVKK) and it looks like we might want to tackle this with an actual property vs using the isBefore. Probably outside of scope for this PR but a headsup.

Reply:
Ooo I like that!
Jordan was also mentioning that we might want to actually have an "archivedAt" since it's apparently something also in the interface (this discussion is happening on that thread as well)